Simplified/NYPLMyBooksDownloadCenter.m

```diff
@@ -266,21 +287,32 @@ - (void)URLSession:(__attribute__((unused)) NSURLSession *)session
         break;
       }
       case NYPLMyBooksDownloadRightsManagementNone: {
-        NSError *error = nil;
-        
+        NSError *removeError = nil, *moveError = nil;
+        NSURL *finalFileURL = [self fileURLForBookIndentifier:book.identifier];
+
         [[NSFileManager defaultManager]
-         removeItemAtURL:[self fileURLForBookIndentifier:book.identifier]
-         error:NULL];
-        
+         removeItemAtURL:finalFileURL
+         error:&removeError];
+
         success = [[NSFileManager defaultManager]
-                   moveItemAtURL:location
```

Comment:
Can't we just use `replaceItemAtURL:withItemAtURL:backupItemName:options:resultingItemURL:error:` instead of having two different operations?

Reply:
good point, probably... but I don't want to change this code too much in the hot fix branch unless I have to. Right now I don't know if this is actually causing any issues.

Reply:
Sure, not an important change anyways